models/enums.js

```diff
@@ -0,0 +1,7 @@
+const orderStatusEnums = Object.freeze({
+  RESERVATE_ON_ORDER_STATUS_ID: 1,
+  CANCELED_ORDER_STATUS_ID: 2,
+  COMPLETED_ORDER_STATUS_ID: 3
+})
```

Comment:
주문(예약) 상태에 대한 데이터를 객체 혹은 enum을 통해 매핑하는 경우 다음과 같이 작성해준다면 차후 사용 시 확장성이 높을 것 같습니다.
```suggestion
const orderStatusEnums = Object.freeze({
  RESERVED: 1,
  CANCELED: 2,
  COMPLETED: 3
})
```